src/xpk/commands/cluster.py

```diff
@@ -279,6 +280,12 @@ def cluster_create(args) -> None:
       xpk_print('Installation of RayCluster failed.')
       xpk_exit(return_code)
 
+  if hasattr(args, 'enable_mtc') and args.enable_mtc:
+    return_code = install_mtc_on_cluster(args, system)
```

Comment:
Hi Abhinav,
The tests are failing - https://screenshot.googleplex.com/7X2ieEgkScXpSTu, do you have an understanding of what's going on with the gcp filestore tests or how they are related?